openshift-kube-apiserver/admission/customresourcevalidation/node/validate_node_config.go

```diff
@@ -103,11 +126,42 @@ func (configNodeV1) ValidateUpdate(_ context.Context, uncastObj runtime.Object,
 	if err := validateConfigNodeForExtremeLatencyProfile(obj, oldObj); err != nil {
 		allErrs = append(allErrs, err)
 	}
+	if err := c.validateMinimumKubeletVersion(obj); err != nil {
+		allErrs = append(allErrs, err)
+	}
 
 	return allErrs
 }
+func (c *configNodeV1) validateMinimumKubeletVersion(obj *configv1.Node) *field.Error {
+	fieldPath := field.NewPath("spec", "minimumKubeletVersion")
+	if !c.waitForSyncedStore(time.After(timeToWaitForCacheSync)) {
+		return field.Forbidden(fieldPath, "caches not synchronized, cannot validate minimumKubeletVersion")
```

Comment:
`field.Internal` would be more appropriate for this:

> This is used to signal that an error was found that was not directly related to user input

Reply:
totally, updated